packages/grafana-ui/src/components/uPlot/plugins/TooltipPlugin2.tsx

```diff
@@ -245,6 +254,10 @@ export const TooltipPlugin2 = ({ config, hoverMode, render, clientZoom = false,
           scheduleRender(true);
         }
       });
+
+      if (sync && sync() === DashboardCursorSync.Crosshair) {
```

Comment:
Not sure if needed here, but should we also add a check to verify that xField is === to time similar to [this recent PR for cursor sync](https://github.com/grafana/grafana/pull/78496)?